src/plugin/cursor.js

```diff
@@ -83,7 +83,6 @@ export default class CursorPlugin {
     };
 
     /**
-     * @private
      * @param {object} e Mouse move event
      */
     _onMousemove = e => {
```

Comment:
the underscore in front of these methods implies it's private. What do you suggest we do with those (without breaking the api)?

Reply:
Underscore not make the method truly private, it can still be accessed outside the body of class, so it can be inherited. But yes, it is a convention and a "suggestion" that is treated as a private method.
So we can:
 - or leave the underscore, because it can be inherited
- or follow the formal conventions, and remove all underscores in this plugin. Since it was considered private, these methods were only used in this class, so the modifications are limited only in this class.

Reply:
+1 for removing the underscores. what about you @katspaugh?

Reply:
I think the _ in this case means that they are functions without `this`. Because they are passed directly as event listener callbacks:
`this.wrapper.addEventListener('mousemove', this._onMousemove);`
So I would keep them prefixed.

Reply:
So, if the underscore in this case don't mean that formally these methods are considered private, they can be leave with this notation, because they can be inherited without problems, correct?

Reply:
>  they can be leave with this notation, 

Agreed.